target/linux/bcm63xx/patches-5.4/569-board_dsl-2750u-c1.patch

```diff
@@ -0,0 +1,68 @@
+--- a/arch/mips/bcm63xx/boards/board_bcm963xx.c
++++ b/arch/mips/bcm63xx/boards/board_bcm963xx.c
+@@ -749,6 +749,49 @@ static struct board_info __initdata boar
+ 	},
+ };
+ 
++static struct board_info __initdata board_dsl_2750u_c1 = {
```

Comment:
Do you really need a full new definition here?

Reply:
Sorry. I'm new to this stuff..can you please elaborate?

Reply:
You say it uses the same hardware as the a4001n. So, why just copy/paste the same definitions again? You could just add the compatible for this device in 528-board_bcm6328-adb-a4001n.patch and use the existing board definition there:

+	{ .compatible = "d-link,dsl-2750u-c1", .data = &board_A4001N, },